.github/workflows/reusable-build-macos.yml

```diff
@@ -0,0 +1,41 @@
+on:
+  workflow_call:
+    inputs:
+      config_hash:
+        required: true
+        type: string
+
+jobs:
+  build_macos:
+    name: Test on macOS
```

Comment:
Is it possible not to fill in the name anyway?
Looks like it changes the display name but if possible I would like to maintain the current display name.

Reply:
Done :)